rules/stylelint-order.js

```diff
@@ -19,246 +18,284 @@ module.exports = {
-          'background',
-          'background-attachment',
-          'background-clip',
-          'background-color',
-          'background-image',
-          'background-origin',
-          'background-position',
-          'background-repeat',
-          'background-size',
-
-          /* cursor */
-          'cursor',
-
-          /* padding */
-          'padding',
-          'padding-top',
-          'padding-right',
-          'padding-bottom',
-          'padding-left',
-
-          /* width */
-          'width',
-          'min-width',
-          'max-width',
-
-          /* height */
-          'height',
-          'min-height',
-          'max-height',
-
-          /* overflow */
-          'overflow',
-          'overflow-x',
-          'overflow-y',
-          'resize',
-
-          /* list-style */
-          'list-style',
-          'list-style-type',
-          'list-style-position',
-          'list-style-image',
-          'caption-side',
-
-          /* tables */
-          'table-layout',
-          'border-collapse',
-          'border-spacing',
-          'empty-cells',
-
-          /* animation */
-          /* animation-[[name] [duration] [timing-function] [delay] [iteration-count] [direction] [fill-mode] [play-state]]*/
-          'animation',
-          'animation-name',
-          'animation-duration',
-          'animation-timing-function',
-          'animation-delay',
-          'animation-iteration-count',
-          'animation-direction',
-          'animation-fill-mode',
-          'animation-play-state',
-
-          /* vertical-aligment */
-          'vertical-align',
-
-          /* text-alignment & decoration */
-          'direction',
-          'tab-size',
-          'text-align',
-          'text-align-last',
-          'text-justify',
-          'text-indent',
-          'text-transform',
-          'text-decoration',
-          'text-decoration-color',
-          'text-decoration-line',
-          'text-decoration-style',
-          'text-rendering',
-          'text-shadow',
-          'text-overflow',
-
-          /* text-spacing */
-          'line-height',
-          'word-spacing',
-          'letter-spacing',
-          'white-space',
-          'word-break',
-          'word-wrap',
-          'color',
-
-          /* font */
-          'font',
-          'font-family',
-          'font-size',
-          'font-size-adjust',
-          'font-stretch',
-          'font-weight',
-          'font-smoothing',
-          'osx-font-smoothing',
-          'font-variant',
-          'font-style',
-
-          /* content */
-          'content',
-          'quotes',
-
-          /* counters */
-          'counter-reset',
-          'counter-increment',
-
-          /* breaks */
-          'page-break-before',
-          'page-break-after',
-          'page-break-inside'
+    'order/properties-order': [ // Concentric CSS order taken from https://github.com/brandon-rhodes/Concentric-CSS
```

Comment:
I was thinking you were planning to move extending [stylelint-config-concentric-order](https://www.npmjs.com/package/stylelint-config-concentric-order) rather than just updating this list

Reply:
So I wanted to, but that package doesn’t seem to work, and it has so few participants that it’s barely maintained. The author also isn’t tagging releases, manually updating the version number in package.json with each update, and it feels too amateur-ish to tie ourselves to the repo/depend on it

Reply:
that makes me a sad panda.